package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "mongo-mongoose",
+  "version": "1.0.0",
+  "description": "",
+  "main": "mongoosePlayground.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch --runInBand",
+    "lint": "eslint .",
+    "pretest": "npm run lint",
+    "start": "nodemon server.js"
+  },
```

```suggestion
  },
  "jest": {
    "testEnvironment": "node"
  },
```
Mongoose recommends setting up Jest configuration for node either by adding the above to package.json or jest.config.js (if you go that route):
https://mongoosejs.com/docs/jest.html